channeldb/invoice_test.go

```diff
@@ -114,7 +114,7 @@ func TestInvoiceWorkflow(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unable to fetch invoice: %v", err)
 	}
-	if dbInvoice2.Terms.State != ContractSettled {
+	if dbInvoice2.State != ContractSettled {
```

Comment:
Commit message describes move of final delta and expiry, but that doesn't happen in this commit anymore

Reply:
good catch, restored